users/src/api/services/user.service.js

```diff
@@ -0,0 +1,61 @@
+const boom = require('@hapi/boom');
+const bcrypt = require('bcrypt');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
const bcrypt = require("bcrypt");
```